composer.json

```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "php": "~5.3|~7.0",
+        "php": "~5.4|~7.0",
```

Comment:
Why php `5.4`?

Reply:
because we use shorttag arrays (already in more code), and my editor said that's not possible in php 5.3 (only 5.4+), so I changed it rather than changing previous code.